app/controllers/stats_controller.rb

```diff
@@ -105,6 +105,16 @@ def comments
     format(data, 'comment')
   end
 
+  def tag
+    data = Tag.select(:tid, :name, :parent, :count).all
+    format(data, 'tag')
+  end
+
+  def node_tag
```

Comment:
```suggestion
  def node_tags
```

Reply:
I'm laying [this one](https://github.com/publiclab/plots2/issues/5412#issuecomment-481769938) on @jywarren  : P  No, no, jk, jk.  It was amazing that he wrote it out for me.  And I should have caught that. I've been looking a lot at the models and controllers lately and there are ones that can be referenced above.  : )

Reply:
Whoops! Forgot to commit it.